start_docker_dev.sh

```diff
@@ -1,5 +1,5 @@
 HSUSER=headerservice
-TAGNAME=4.0.0-5.0.0-salobj_5.8.0-aths_2.0.0
+TAGNAME=4.1.0-5.0.0-salobj_5.10.0-aths_2.0.0
```

Comment:
Is the ATHS tag version updated just before a new release is done?

Reply:
Yes, usually after the PR is done. But I should change `aths` to just `hs` or `headerservice`